misc/factor.vim.fgen

```diff
@@ -1,333 +1,98 @@
-<%
-USING: accessors io kernel multiline prettyprint sequences sorting vocabs ;
+USING: accessors
+    calendar calendar.format
+    io kernel multiline prettyprint sequences sorting
+    splitting tr vocabs ;
 IN: factor.vim.fgen
+ALIAS: wr write
```

Comment:
Why not just use ``write``?  Is there a name conflict somehow?

Reply:
It's used a lot, and helps keep a non-trivial amount of lines `textwidth=64` compliant.